src/app/garfile/view/post.controller.js

```diff
@@ -6,7 +6,10 @@ const garApi = require('../../../common/services/garApi');
 module.exports = (req, res) => {
   logger.debug('In garfile / view post controller');
   const cookie = new CookieModel(req);
-  const { garId } = req.body;
+  let { garId } = req.body;
```

Comment:
The GAR Summary page takes a POST call with "garId" expected to be in the form submitted.

Update now falls back to checking the session (via the cookie.getGarId()) if this is not set, so that the aforementioned 307 redirect does not require a POST submit with the GAR ID set, which would be derived from the cookie.getGarId() call anyway.